docs/index.rst

```diff
@@ -39,16 +45,7 @@ Links
 
 License: BSD-3
 
-.. _astroplan_news:
 
-News
-====
-
-* August 28, 2015: Astroplan **0.1** release
```

Comment:
Well, that's set by the version number in the setup files, right?  It also appears in the sidebar and in the URL on readthedocs, or on github it appears as the latest tag.  I don't like the idea of hard-coding the version number in multiple places... More generally I take a dim view of having anything like "news" or other items that must be updated on a timely basis on the documentation page: these things are nearly *always* stale. Better to let the automated tools reflect the actual version information.